modules/swagger-codegen/src/main/resources/Java/libraries/retrofit2/build.gradle.mustache

```diff
@@ -94,9 +94,8 @@ if(hasProperty('target') && target == 'android') {
 }
 
 ext {
-    oltu_version = "1.0.1"
-    retrofit_version = "2.0.0-beta4"
-    gson_version = "2.6.2"
+    oltu_version = "1.0.0"
```

Comment:
Yes please. We want to upgrade the dependencies to the latest version after stable releases.